### PR TITLE
fix: small fixes for different bugs

### PR DESCRIPTION
**Pull request by davidsoderberg**
### What change does this PR introduce? 
Small fixes for reported bugs in template editor

### CODE REVIEW

File: apps/api/src/app/events/usecases/send-message/test-send-email.usecase.ts [hunk 1]
```diff
@@ -58,10 +58,24 @@ export class SendTestEmail {
...
+          </div>
+        {{/if}}$&`
+      );
+    }
```
Comment (scopsy) [inline apps/api/src/app/events/usecases/send-message/test-send-email.usecase.ts:73]: Could we add a test case or a couple for this functionality? We can do a small unit test for it.

File: apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts [hunk 2]
```diff
@@ -185,10 +185,24 @@ export class SendMessageEmail extends SendMessageType {
...
...&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;
```
Comment (scopsy) [inline apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts:196]: Why is this line needed?

**Review by scopsy — COMMENTED** [review]
Looks good, missing some tests for the new preheader functionality and a few small questions

File: apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts (hunk 2 above)
Comment (davidsoderberg) [reply apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts:196]: This is the soultion I found online to not get the content from the email after the preheader in the preheader area in gmail or other email clients 😄
